container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -241,9 +246,54 @@ instance_groups:
       properties.login.smtp.starttls: ((SMTP_STARTTLS))
       properties.login.smtp.user: ((SMTP_USER))
       properties.uaa.logging_level: ((LOG_LEVEL_LOG4J))((#LOG_LEVEL))((/LOG_LEVEL))
-      properties.uaadb.roles: '[{"name": "uaaadmin", "password": "((UAADB_PASSWORD))", "tag": "admin"}]'
-      properties.wait-for-database.hostname: mysql-proxy-set
-      properties.wait-for-database.port: 3306
+      properties.uaadb.address: &cf-internal-database-host >
+        ((#DB_EXTERNAL_HOST))
+          ((DB_EXTERNAL_HOST))
+        ((/DB_EXTERNAL_HOST))
+        ((^DB_EXTERNAL_HOST))
+          mysql-proxy-set.((KUBERNETES_NAMESPACE)).svc.((KUBERNETES_CLUSTER_DOMAIN))
+        ((/DB_EXTERNAL_HOST))
+      properties.uaadb.db_scheme: &cf-internal-database-scheme >
```

Comment:
I don't think `cf-internal-database-scheme` was ever reused (the anchor doesn't need to be here).

Reply:
Confirmed. Will fix.

Reply:
Done with commits 7d897b5a65dc4e5af7d21cf30d83e583a020ed21 and SUSE/scf@addaf660ee8dc5dd16d6def77b307d731dcdbd31